.github/scripts/test_behavior/plan.py

```diff
@@ -198,6 +198,10 @@ def generate_language_binding_cases(
 ) -> list[dict[str, str]]:
     cases = unique_cases(cases)
 
+    # Remove hdfs cases for java.
+    if language == "java":
+        cases = [v for v in cases if v["service"] != "hdfs"]
```

Comment:
Actually, a cleaner solution is following Python binding's tests to skip unknown service tests.

Please create an issue and I'll follow up.